src/libraries/Microsoft.Extensions.DependencyInjection/ref/Microsoft.Extensions.DependencyInjection.Forwards.cs

```diff
@@ -1,7 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// ------------------------------------------------------------------------------
-// Changes to this file must follow the https://aka.ms/api-review process.
-// ------------------------------------------------------------------------------
-
-[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(Microsoft.Extensions.DependencyInjection.ServiceCollection))]
```

Comment:
Can we ensure this isn't broken?

Reply:
Yes, the type forward is present in the generatedreference assembly:

![image](https://user-images.githubusercontent.com/7412651/155404305-b92919af-f59f-41c9-943f-8eca9a5af33a.png)

I don't know if that's possible but would it make sense to teach GenAPI to include typeforwards in the ref file?

Reply:
> I don't know if that's possible but would it make sense to teach GenAPI to include typeforwards in the ref file?

Considering this is how we verify public API changes, I think it should include TypeForwardedTo.